File: src/FE/jts/src/components/statistics/tournament-view.js
import { React } from 'react';

const TournamentView = ({ index, tournamentId, when }) => {
    return (
        <tr>
            <td>{index}</td>
            <td>{tournamentId}</td>
            <td>{when}</td>
        </tr>
    )
};

export default TournamentView;

File: src/BE/Domain/Statistics/Models/TournamentView.cs
namespace JTSystem.Domain.Statistics.Models
{
    using JTSystem.Domain.Common.Models;
    using System;

    public class TournamentView : Entity<int>
    {
        internal TournamentView(int tournamentId)
        {
            this.TournamentId = tournamentId;
            this.When = DateTime.UtcNow;
        }

        public int TournamentId { get; }

        public DateTime When { get; }
    }
}
